go.mod

```diff
@@ -90,7 +90,7 @@ require (
 	k8c.io/kubelb v1.1.2
 	k8c.io/kubeone v1.7.3
 	k8c.io/machine-controller v1.60.1
-	k8c.io/operating-system-manager v1.6.1-0.20241118134103-5db575f65108
+	k8c.io/operating-system-manager v1.6.1
```

Comment:
A release against 1.6.1 doesn't exist? https://github.com/kubermatic/operating-system-manager/releases

Reply:
Yeah, I think @xrstf will create one soon

Reply:
Ah thanks for reminding us of the OSP bump, I totally forgot that :grimacing:  Will ship OSM 1.6.2.